templates/templates/base.html

```diff
@@ -43,7 +43,12 @@
     <!-- <![endif]-->
 
     <!-- javascript -->
-    <script src="{{ ASSET_SERVER_URL }}b0601acd-modernizr.2.7.1.js"></script>
+    {% include "shared/_html5_shiv.html" %}
+    <!-- replace no-js -->
+    <script>
+      var html = document.documentElement
+      html.className = html.className.replace(/\bno-js\b/, 'js')
```

Comment:
You could use classList.replace here https://developer.mozilla.org/en-US/docs/Web/API/Element/classList

Reply:
Done